server/app/src/androidTest/java/sh/calaba/instrumentationbackend/actions/version/Version.java

```diff
@@ -13,10 +13,10 @@
 
 	/******
 	 * Generate Version Number
-	 * DO NOT EDIT
+	 * DO NOT EDIT THIS FILE MANUALLY
 	 *
 	 */
-	public static final String VERSION="####VERSION####";
+	public static final String VERSION="0.9.2";
```

Comment:
?! Do we intend to have a hardcoded version committed to git?

Reply:
This version will be updated during build

Reply:
task replaceVersion will update this. Anyway this version can't break anything and the fact, that we have version in source files allows us to get rid of "staging" folder